dev.mariinkys.Oboete.json

```
@@ -0,0 +1,49 @@
+{
+    "app-id": "dev.mariinkys.Oboete",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "oboete",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "modules": [
+        {
+            "name": "oboete",
+            "buildsystem": "simple",
+            "build-options": {
+                "env": {
+                    "CARGO_HOME": "/run/build/oboete/cargo"
+                }
+            },
+            "build-commands": [
+                "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+                "cargo --offline build --release --verbose",
+                "install -Dm0755 ./target/release/oboete /app/bin/oboete",
+                "install -Dm0644 ./res/dev.mariinkys.Oboete.desktop /app/share/applications/dev.mariinkys.Oboete.desktop",
+                "install -Dm0644 ./res/dev.mariinkys.Oboete.metainfo.xml /app/share/metainfo/dev.mariinkys.Oboete.metainfo.xml",
+                "install -Dm0644 ./res/icons/hicolor/256x256/apps/dev.mariinkys.Oboete.png /app/share/icons/hicolor/256x256/apps/dev.mariinkys.Oboete.png",
+                "install -Dm0644 ./res/icons/hicolor/256x256/apps/dev.mariinkys.Oboete.svg /app/share/icons/hicolor/scalable/apps/dev.mariinkys.Oboete.svg"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "path": "https://github.com/mariinkys/Oboete",
+                    "tag": "0.1",
+                    "commit": "7c684847aeb8c9af66dbcb134284ec16803a2502",
+                    "builddir": true
```

```suggestion
                    "commit": "7c684847aeb8c9af66dbcb134284ec16803a2502"
```
`builddir` doesn't belong here.